github-actions/run-clang-format/action.yaml

```
@@ -19,8 +19,9 @@ inputs:
     default: 'always'
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/everest/everest-clang-format:v1.1.0'
+  image: 'docker://ghcr.io/everest/everest-ci/build-env-base:refactor-docker-images' # LTODO: Update this
```

```suggestion
  image: 'docker://ghcr.io/everest/everest-ci/build-env-base:v1.3.0
```

TODO after merge: release `v1.3.0`